deps/Makefile

```diff
@@ -36,7 +36,7 @@ libssl: libssl/openssl/libssl.a
 
 libhttpserver/libhttpserver/build/src/.libs/libhttpserver.a: libmicrohttpd/libmicrohttpd/src/microhttpd/.libs/libmicrohttpd.a
 	cd libhttpserver && rm -rf libhttpserver-master_20191121 || true
-	cd libhttpserver && tar -zxf libhttpserver-master_20191121.tar.gz
+	cd libhttpserver && tar -C libhttpserver -zxf libhttpserver-master_20191121.tar.gz && mv libhttpserver-master_20191121 libhttpserver
```

Comment:
The next line assumes that there's a directory called `libhttpserver`, but the tar is unpacked into `libhttpserver-master_20191121`. It has to be renamed, otherwise the next step fails.